x/incentives/keeper/distribute.go

```diff
@@ -177,6 +188,7 @@ func newDistributionInfo() distributionInfo {
 	}
 }
 
+// TODO: Slightly confused here, how does this determine which lockID to add rewards to? Cant a single owner own multiple locks?
```

Comment:
TODO: Slightly confused here, how does this determine which lockID to add rewards to? Cant a single owner own multiple locks?

Reply:
Ah I didn't catch that, thank you!